src/test/java/core/basesyntax/strategy/FruitImplemDaoTest.java

```diff
@@ -0,0 +1,67 @@
+package core.basesyntax.strategy;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.fail;
+
+import core.basesyntax.dao.DaoFruit;
+import core.basesyntax.dao.FruitImplemDao;
+import core.basesyntax.db.Storage;
+import org.junit.After;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class FruitImplemDaoTest {
+    private static DaoFruit fruitDao;
+
+    @BeforeClass
+    public static void beforeClass() {
+        fruitDao = new FruitImplemDao();
+    }
+
+    @Test
+    public void addFruit_Ok() {
+        fruitDao.addFruits("banana",17);
+        fruitDao.addFruits("grapes",10);
+        fruitDao.addFruits("strawberry",13);
+        long actual = fruitDao.getAmountOfFruit("banana");
+        assertEquals(17, actual);
+    }
+
+    @Test
+    public void getFruitAmount_absentFruit_NotOk() {
+        fruitDao.addFruits("banana",17);
+        fruitDao.addFruits("grapes",10);
+        fruitDao.addFruits("strawberry",13);
+        try {
+            fruitDao.getAmountOfFruit("pink");
+        } catch (Exception e) {
+            assertEquals("There is no pink in storage",
+                    e.getMessage());
+            return;
+        }
+        fail("Expected validation exception was not thrown");
+    }
+
+    @Test
```

Comment:
it is good practice to write your solution consistently. If you use (expected = SomeException.class)  use it everywhere